src/index.js

```diff
@@ -48,9 +48,9 @@ const components = {
   tipBox,
   tooltip,
   pic,
-  trigger,
+	searchbar,
```

Comment:
The indentation here is tab right than space. I think your IDE might be set to tabs rather than space?